### PR TITLE
Quality Of Life Styling

### PR DESCRIPTION
**Pull request by rohan-bansal**
I added a universal font (Lato), reorganized the css, and reorganized all the files into folders that make the workflow easier.

### CODE REVIEW

File: client/src/components/WorkspacePanels/MainWorkspacePanel/MainWorkspace.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.panel {
+  /* Main editing window styling */
+}
```
Comment (nayeonshin) [inline client/src/components/WorkspacePanels/MainWorkspacePanel/MainWorkspace.css]: For now, can you comment out this entire thing?

File: client/src/routes/WorkspaceRoute/Workspace.css [hunk 2]
```diff
@@ -34,6 +35,8 @@ body {
 
 .wrapper__column:nth-child(2) {
   width: 60vw;
+  border-top: 6px solid #02feff;
+  border-bottom: 6px solid #02feff;
```
Comment (nayeonshin) [inline client/src/routes/WorkspaceRoute/Workspace.css:39]: If `6px solid #02feff` appears multiple times (more than like 3 times), you can make it like a variable by writing:
```
:root {
--main-border: 6px solid #02feff;
}
```
And then write smth like `border-top: var(--main-border);` instead.
Ignore this if this only appears twice here in this file.

**Review by nayeonshin — CHANGES_REQUESTED** [review]

File: client/src/components/WorkspacePanels/MainWorkspacePanel/MainWorkspace.css (hunk 1 above)
Comment (rohan-bansal) [reply client/src/components/WorkspacePanels/MainWorkspacePanel/MainWorkspace.css]: Sure!

File: client/src/routes/WorkspaceRoute/Workspace.css (hunk 2 above)
Comment (rohan-bansal) [reply client/src/routes/WorkspaceRoute/Workspace.css:39]: this only appears twice, but thanks for the suggestion!

**Pull request merged**